PyInstaller/hooks/hook-webview.py

```diff
@@ -0,0 +1,24 @@
+"""
+pywebview requires WebBrowserInterop dll on Windows
+"""
+
+import platform
+import ctypes
+
+from PyInstaller.compat import is_win, getsitepackages
+from os.path import join, exists
+
+if is_win:
+    dll_name = 'WebBrowserInterop.x64.dll' if platform.architecture()[0] == '64bit' else 'WebBrowserInterop.x86.dll'
+
+    library = ctypes.util.find_library(dll_name)
+    datas = []
+    if library:
+        datas = [(library, '.')]
+    else:
+    	for sitepack in getsitepackages():
```

Comment:
Please check intention for the whole file. Should be four spaces per level.

Reply:
It's even worse: indentation contains mixed spaces and tabs, see https://travis-ci.org/pyinstaller/pyinstaller/jobs/434296736